proto/brpb.proto

```diff
@@ -898,6 +898,48 @@ message MetaEdit {
     bool destruct_self = 4;
 }
 
+// RewrittenTableID records a sort of modification over the SSTs during restoring.
+// 
+// When "backing up" this "restored" SST, we want to backup what it was "restored".
+// But in some cases, SSTs are not restored "as is", TiKV may rewrite or truncate its
+// content by the client's request. 
+message RewrittenTableID {
+    // The table ID in the backup data.
+    int64 upstream_of_upstream = 1;
+    // The rewritten table ID during restoring.
+    int64 upstream = 2;
```

Comment:
```suggestion
    int64 table_id_in_ancestor_upstream = 1;
    // The rewritten table ID during restoring.
    int64 table_id_in_upstream_table_id = 2;
```

Reply:
agree, but we may need to explain what `ancestor` is, as it's a new term.

Reply:
It seems there are too many repeated `table_id`s. I think `ancestor_upstream` and `upstream` would be fine.